pkg/pillar/cmd/ledmanager/ledmanager.go

```diff
@@ -57,13 +59,18 @@ type DisplayFunc func(deviceNetworkStatus *types.DeviceNetworkStatus,
 // InitFunc takes an argument which can be the name of a LED or display
 type InitFunc func(arg string)
 
+// AppStatusDisplayFunc takes an argument to list of leds
+type AppStatusDisplayFunc func(blinkCh chan string, arg []string, blink bool, color string)
+
 type modelToFuncs struct {
-	model       string
-	initFunc    InitFunc
-	displayFunc DisplayFunc
-	arg         string // Passed to initFunc and displayFunc
-	regexp      bool   // model string is a regex
-	isDisplay   bool   // no periodic blinking/update
+	model                string
+	initFunc             InitFunc
+	displayFunc          DisplayFunc
+	arg                  string // Passed to initFunc and displayFunc
+	appStatusDisplayFunc AppStatusDisplayFunc
+	leds                 []string // Passed to appStatusDisplayFunc
```

Comment:
If we have an LCD to display on then this argument  might not be leds. So something like appStatusArgs might be a better name.

Reply:
Sure will change that.